images/macos/provision/core/homebrew.sh

```diff
@@ -7,14 +7,15 @@ HOMEBREW_INSTALL_URL="https://raw.githubusercontent.com/Homebrew/install/master/
 echo "Disabling Homebrew analytics..."
 brew analytics off
 
-echo "Installing the latest curl..."
+# jq is required for further installation scripts
+echo "Install jq"
+brew install jq
+
+echo "Install curl"
 brew install curl
 
-echo "Installing wget..."
+echo "Install wget"
 brew install wget
 
-echo "Installing jq..."
-brew install jq
-
 # init brew bundle feature
 brew tap Homebrew/bundle
```

Comment:
Race condition here, we cannot start tests since powershell is not installed on this step, but we cannot make powershell first since for powershell jq and brew are required.